config/webpack.dev.config.js

```diff
@@ -49,12 +49,17 @@ module.exports = {
         postcss: [autoprefixer, precss],
       }
     })
-  ],
-  resolve: {
+  ]
+}
+
+if (process.env.BABEL_ENV === 'TEST') {
+  devConfigs.revolve = {
```

Comment:
this fix isn't right.
the reason why it is working is because ```resolve``` is spelled wrongly.
you can actually jus remove the resolve block as these are npm module if i am not wrong.